wsgi: always record span status code to have it available in metrics (#3148)
open-telemetry/opentelemetry-python-contrib: CHANGELOG.md

  ([#3111](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/3111))
- `opentelemetry-instrumentation-falcon` add support version to v4
  ([#3086](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/3086))

> - `opentelemetry-instrumentation-wsgi` remove is_recording method to record status code in all metrics

Reviewer: ```suggestion
- `opentelemetry-instrumentation-wsgi` always record span status code to have it available in metrics
```
